.tool-versions

```diff
@@ -0,0 +1,3 @@
+maven 3.9.1
```

Comment:
if we add this version here, must be equal to this one: https://github.com/kuflow/kuflow-samples-java/blob/main/.mvn/wrapper/maven-wrapper.properties